deps/build.jl

```diff
@@ -1,90 +1,91 @@
     info("Backing up existing config file to $backup_file")
     Playground.copy(config_file, backup_file)
 end
 
-fstream = open(config_file, "w+")
-write(fstream, Playground.DEFAULT_CONFIG)
-close(fstream)
+open(config_file, "w+") do fstream
+    write(fstream, Playground.DEFAULT_CONFIG)
+end
 
 @unix_only begin
-    Playground.copy(joinpath(deps_path, "usr/bin/INSTALL.sh"), joinpath(build_path, "INSTALL.sh"))
+    Playground.copy(joinpath(deps_dir, "usr", "bin", "INSTALL.sh"), joinpath(build_dir, "INSTALL.sh"))
 end
 
-Playground.copy(joinpath(deps_path, "../LICENSE"), joinpath(build_path, "LICENSE"))
-Playground.copy(joinpath(deps_path, "../README.md"), joinpath(build_path, "README.md"))
+Playground.copy(joinpath(deps_dir, "..", "LICENSE"), joinpath(build_dir, "LICENSE"))
+Playground.copy(joinpath(deps_dir, "..", "README.md"), joinpath(build_dir, "README.md"))
 
 # Only install the config and executable if the
 # PLAYGROUND_INSTALL env variable has been set.
 # This is just cause there isn't a `Pkg.install` or
 # `Pkg.build("Pkg", install=true)`
 install = haskey(ENV, "PLAYGROUND_INSTALL")
+
 # Store our install paths
-install_path = Playground.config_path()
-install_config = joinpath(install_path, "config.yml")
-install_playground_exec = joinpath(install_path, "bin", "playground")
+install_dir = Playground.config_path()
+config_installed = joinpath(install_dir, "config.yml")
+playground_installed = joinpath(install_dir, "bin", "playground")
+playground_compiled = joinpath(build_dir, "playground")
 
 if install
     # Set up the user level playground directory
     info("Setting up user playground directory...")
-    mkpath(install_path)
-    mkpath(joinpath(install_path, "bin"))
+    mkpath(install_dir)
+    mkpath(joinpath(install_dir, "bin"))
 
-    info("Linking playground config to $install_path/config.yml.")
+    info("Linking playground config to $config_installed.")
 
     if ispath(install_config)
         info("~/.playground/config.yml already exists. Skipping.")
         info("Please see $config_file if you have any problems with your existing config.yml file.")
     else
-        Playground.mklink(config_file, install_config)
+        symlink(config_file, install_config)
     end
 
-    info("Linking playground executable to $install_playground_exec")
+    info("Linking playground executable to $playground_installed")
 
-    if ispath(install_playground_exec)
-        backup_file = joinpath(install_path, "bin", ".playground_$(Dates.today()).bak")
+    if ispath(playground_installed)
+        backup_file = joinpath(install_dir, "bin", ".playground_$(Dates.today()).bak")
         info("Backing up existing playground executable to $backup_file")
-        Playground.copy(install_playground_exec, backup_file)
+        Playground.copy(playground_installed, backup_file)
     end
 
-    Playground.mklink(joinpath(build_path, "playground"), install_playground_exec)
+    symlink(playground_compiled, playground_installed)
```

Comment:
This will throw an error on repeat installs which is why I was using `mklink` as it will by default overwrite the existing link. I can't decide which I prefer.